app/cypress/e2e/cif/project-revision/index.cy.js

```diff
@@ -217,16 +217,13 @@ describe("the new project page", () => {
       "$1.00",
       "50.00 %",
       "$777.00",
-      "99.87%",
+      "This field cannot be calculated due to lack of information now.",
       /Jun(\.)? 10, 2020/,
       /Jun(\.)? 10, 2020/,
       "$1.00",
-      "$778.00",
+      "This field cannot be calculated due to lack of information now.",
```

Comment:
I agree that we need to test that the fields are showing correctly when not enough information is entered, but is this removing confirming that the funding source is showing correctly when values are entered from the e2e tests?

Reply:
The funding source calculations should handle this, and I've added more db unit tests for better coverage.